diff --git a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
index 7dbddf5..20ed99f 100644
--- a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
+++ b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
@@ -94,6 +94,8 @@ exports.networks = [
     { chainId: 12357, shortName: 'rei-testnet' },
     { chainId: 23294, shortName: 'sapphire' },
     { chainId: 23295, shortName: 'sapphire-testnet' },
+    { chainId: 32769, shortName: 'zil' },
+    { chainId: 33101, shortName: 'zil-testnet' },
     { chainId: 42161, shortName: 'arb1' },
     { chainId: 42170, shortName: 'arb-nova' },
     { chainId: 42220, shortName: 'celo' },
