diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index a3c670d..e69ae16 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -224,7 +224,9 @@ exports.networks = [
     { chainId: 1666700000n, shortName: 'hmy-b-s0' },
     { chainId: 11297108099n, shortName: 'tpalm' },
     { chainId: 11297108109n, shortName: 'palm' },
-    { chainId: 88153591557n, shortName: 'arb-blueberry' }
+    { chainId: 88153591557n, shortName: 'arb-blueberry' },
+    { chainId: 32769n, shortName: 'zil' },
+    { chainId: 33101n, shortName: 'zil-testnet' },
 ];
 if (process.env.TEST_NETWORK === 'hardhat') {
     exports.networks.push({ shortName: 'local', chainId: 31337n });
